lib/app.tsx

```diff
@@ -30,7 +30,10 @@ import {
   pick,
   values,
 } from 'lodash';
-import { toggleSimperiumConnectionStatus } from './state/ui/actions';
+import {
+  setUnsyncedNoteIds,
+  toggleSimperiumConnectionStatus,
+} from './state/ui/actions';
```

Comment:
should be able to import them all in one go if you find it easier

```ts
import * as actions from './state/actions';

dispatch( actions.ui.setUnsyncedNoteIds );
```

Reply:
I like having them separated. If there were more than five or we had multiple /actions imports this would make sense to me.